Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sealed"
-version = "0.1.3"
+version = "0.2.0-rc1"
```

Comment:
Also, it would be nice to swith to `resolver = "2"` for Cargo, and use `default-features = false` for main dependencies, enabling only those, which are really required.

Reply:
FWIW, `resolver = "2"` doesn't really matter for a simple project layout such as this one, since it will act as _a dependency_ for a downstream workspace; where it matters is for users of that workspace, especially `no_std` users, for whom using `resolver = "2"` may allow to avoid breakage if they happen to share a dependency with the ones used here (_e.g._, `::serde` through `::proc_macro_crate`), and if the dependency _here_ enables a `no_std`-incompatible feature (if I am saying all this is also that I am actually realizing it may finally be time to start using again `::proc_macro_crate`, now that users have a way to solve that bug 🙂).

  - `default-features = "false"` is a good approach to have, yes (although I'm pretty sure for the case of `syn`, `quote`, and `proc-macro2`, all the _current_ default features are required).